stubs/fpdf2/fpdf/fpdf.pyi

```diff
@@ -128,9 +137,11 @@ class FPDF:
         orientation: _Orientation = ...,
         unit: _Unit | float = ...,
         format: _Format | tuple[float, float] = ...,
-        font_cache_dir: bool = ...,
+        font_cache_dir: bool | Literal["DEPRECATED"] = ...,
```

Comment:
The docstring says "path or str". But it's also deprecated and ignored at runtime, so it doesn't really matter much.

https://github.com/PyFPDF/fpdf2/blob/be3ad07b7ae9c7d39866aa130de177b43fc3f073/fpdf/fpdf.py#L329